job-dsls/jobs/prod/prod_bamoe_properties_generator.groovy

```diff
@@ -50,8 +50,8 @@ pipelineJob("${folderPath}/bamoe-properties-generator") {
     description("Generate properties files for BAMOE productized builds")
 
     parameters {
-        stringParam("REPO_URL", "http://download.devel.redhat.com/rcm-guest/staging")
-        stringParam("DELIVERABLE_REPO_URL", "http://download.devel.redhat.com/devel/candidates/middleware")
+        stringParam("REPO_URL", "\${env.STAGING_SERVER_URL}")
+        stringParam("DELIVERABLE_REPO_URL", "\${env.DOWNLOAD_CANDIDATES}/middleware")
```

Comment:
env variables can't be used here, please move this value inside the job itself

Reply:
according to https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/custom/job/emingora/job/envtest/2/console it can be used, so I agree on leave it as it is.